src/signals/incident-management/containers/IncidentOverviewPage/styled.tsx

```diff
@@ -81,10 +83,8 @@ export const NavWrapper = styled(Column).attrs({
   margin-bottom: ${themeSpacing(5)};
 `
 
-export const PageTitle = styled.div`
-  display: flex;
-  justify-content: space-between;
-  margin-bottom: ${themeSpacing(2)};
+export const StyledBackLink = styled(BackLink)`
+  margin-top: ${themeSpacing(6)};
```

Comment:
in line with UX, instead of 22 px from figma we align with margins in incident detail.